src/gatling/resources/bodies/caseManagement/assignCase.json

```diff
@@ -0,0 +1,5 @@
+{
+  "assignee_id": "${AssignmentIdamId}",
```

Comment:
A bit fussy, but could you make these consistent between the files (i.e. camelCase)?

Reply:
Absolutely.  I have updated these in latest commit.  I have kept Username and Password the same case as they are both used in existing repo Idam authentication that requires this case.